src/Components/Components/src/RenderTree/RenderTreeFrame.cs

```diff
@@ -48,153 +48,181 @@ namespace Microsoft.AspNetCore.Components.RenderTree
         // Common
         // --------------------------------------------------------------------------------
 
+        [FieldOffset(0)] internal int SequenceField;
+        [FieldOffset(4)] internal RenderTreeFrameType FrameTypeField;
```

Comment:
Is the goal that we don't "break" the entire world with this change?

Reply:
Yes, this change won't be breaking for any code that either (a) is recompiled after this change goes in, or (b) doesn't use `RenderTreeFrame` directly (like our analyzer tells people).

Reply:
I'm thinking of things like bUnit and so on, they'll likely ignore your warning :)

Reply:
Yes I strongly expect so. I was going to contact @egil about this directly.